apps/api/src/config/index.ts

```diff
@@ -160,13 +160,18 @@ export const config = {
   },
   // Frontend app configuration for interfacing with the server
   app: {
+    // Session management and email verification settings for the frontend competitions app
     url: process.env.FRONTEND_URL || "http://localhost:3001", // TODO: resolve frontend/backend default ports
     domain: process.env.DOMAIN,
     cookieName: process.env.COOKIE_NAME || "session",
     sessionPassword:
       process.env.ROOT_ENCRYPTION_KEY ||
       "default_encryption_key_do_not_use_in_production",
     sessionTtl: parseInt(process.env.SESSION_TTL || "7200", 10),
+    // CORS origins for API calls (e.g., frontend app, docs site + OpenAPI UI, etc.)
+    corsOrigins: process.env.CORS_ORIGINS?.split(",") || [
```

Comment:
new env var that we want to set in prod is a comma separate list of URLs: `CORS_ORIGINS`

if it's not set, we'll just default to the FRONTEND_URL or localhost defaults